cli/command/new/templates.go

```diff
@@ -1,74 +1,142 @@
 package new
 
-var configTpl = `apiVersion: v1alpha
-metadata:
-  name: Feature XYZ
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - {{ .TestsFolder }}/example.test.js
-suites:
-  - name: "saucy test"
-    match: "{{ .Match }}"
-image:
-  base: {{ .Name }}
-  version: {{ .Version }}
-sauce:
-  region: {{ .Region }}
-`
-
-// SetupTemplate describes a template for a setup
-type SetupTemplate struct {
-	Filename string
-	Code     string
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"github.com/google/go-github/github"
+	"github.com/rs/zerolog/log"
+	"github.com/tj/survey"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+var (
+	overWriteAll     = false
```

Comment:
Why global? As far as I can see, it's only used in `func confirmOverwriting()`, so might be better to just add to the function args?

Reply:
I was considering make it available to multiple parts (and not just for extracting templates), but removed this.
I removed this to carry through function args.